src/main/scala/com/databricks/labs/overwatch/utils/Upgrade.scala

```diff
@@ -310,29 +310,32 @@ object Upgrade extends SparkSessionWrapper {
    * @param pipelineReportPath pass in database name
    * @return
    */
-  private def getLatestWorkspaceByOrg(pipelineReportPath: String): Array[OrgConfigDetail] = {
+  def getLatestWorkspaceByOrg(pipelineReportPath: String): Array[OrgConfigDetail] = {
     val oldestDateByModuleW = Window.partitionBy('organization_id).orderBy('Pipeline_SnapTS.desc)
 
-    // handle optional nulls error when building OverwatchParams
-    val addNewConfigs = Map(
-      "inputConfig.auditLogConfig.azureAuditLogEventhubConfig" ->
-        when($"inputConfig.auditLogConfig.rawAuditPath".isNotNull, lit(null))
-          .otherwise($"inputConfig.auditLogConfig.azureAuditLogEventhubConfig")
-          .alias("azureAuditLogEventhubConfig")
-    )
-
     val overwatchParamsCols = Array("auditLogConfig", "tokenSecret", "dataTarget", "badRecordsPath", "overwatchScope",
       "maxDaysToLoad", "databricksContractPrices", "primordialDateString", "intelligentScaling", "workspace_name",
       "externalizeOptimize")
 
     // Necessary because if recovering pipeline upgrade the columns may or may not exist so both scenarios
     // must be handled. This ensures both columns are present and present only once
-    val orgRunDetailsBase = spark.read.format("delta").load(pipelineReportPath)
+    val orgRunDetailsBase = spark.read.format("delta").load("/mnt/overwatch_global/05x/global_share/pipeline_report")
```

Comment:
Is it correct to hardcore path?

Reply:
nope that's a booboo. Customer noticed too lol dangit.